### PR TITLE
[L02] Inconsistent quorum calculation

### PR DESCRIPTION
**Pull request by mrsmkl**
### Related issues

- Fixes https://github.com/celo-org/celo-labs/issues/302

### CODE REVIEW

File: packages/celotool/src/e2e-tests/transfer_tests.ts [hunk 1]
```diff
@@ -83,7 +83,8 @@ const INTRINSIC_TX_GAS_COST = 21000
 // Additional intrinsic gas for a transaction with fee currency specified
 const ADDITIONAL_INTRINSIC_TX_GAS_COST = 50000
 
-const stableTokenTransferGasCost = 20653
+// const stableTokenTransferGasCost = 20653
```
Comment (asaj) [inline packages/celotool/src/e2e-tests/transfer_tests.ts]: Why is this different?

File: packages/protocol/contracts/common/UsingPrecompiles.sol [hunk 2]
```diff
@@ -234,4 +234,21 @@ contract UsingPrecompiles {
     }
     return x;
   }
+
+  /**
+   * @notice Returns the minimum number of required signers for a given block number.
+   * @dev Computed in Geth as int(math.Ceil(float64(2*valSet.Size()) / 3))
```
Comment (asaj) [inline packages/protocol/contracts/common/UsingPrecompiles.sol]: nit: celo-blockchain

File: packages/celotool/src/e2e-tests/transfer_tests.ts (hunk 1 above)
Comment (mrsmkl) [reply packages/celotool/src/e2e-tests/transfer_tests.ts]: This sometimes changes, haven't found the reason yet...

File: packages/protocol/contracts/common/UsingPrecompiles.sol [hunk 3]
```diff
@@ -234,4 +234,21 @@ contract UsingPrecompiles {
...
+   */
+  function minQuorumSize(uint256 blockNumber) internal view returns (uint256) {
+    return numberValidatorsInSet(blockNumber).mul(2).add(2).div(3);
+  }
+
+  /**
+   * @notice Computes byzantine quorum from current validator set size
+   * @return Byzantine quorum of validators.
+   */
+  function byzantineQuorumValidatorsInCurrentSet() public view returns (uint256) {
```
Comment (nategraf) [inline packages/protocol/contracts/common/UsingPrecompiles.sol]: Could you rename one of these two to be match whether `min` or `byzantine` is used at the start?

File: packages/celotool/src/e2e-tests/transfer_tests.ts (hunk 1 above)
Comment (nategraf) [reply packages/celotool/src/e2e-tests/transfer_tests.ts]: +1 to this. We need to make sure we know why gas costs change.

Comment (mrsmkl) [reply packages/celotool/src/e2e-tests/transfer_tests.ts]: It was because a public method was added to UsingPrecompiles, so the routing code changed.

**Review by asaj — APPROVED** [review]

**Pull request merged**